src/generic/key-utils.ts

```diff
@@ -126,3 +126,18 @@ export function normalizeContainerType(containerType: ContainerType | string) {
       return containerType;
   }
 }
+
+/**
+ * Given a usage key of V1 block like `block-v1:org+type@html+block@1`, get the type (e.g. `html`)
+ * @param usageKey e.g. `block-v1:org+type@html+block@1`
+ * @returns The block type as a string
+ */
+export function getBlockTypeBlockV1(usageKey: string): string {
```

Comment:
Why don't you combine this into the existing `getBlockType` function? I think it's fine to have one function that handles both library keys and modulestore course block keys. People can use `isBlockV1Key` explicitly in other places to enforce a key type.